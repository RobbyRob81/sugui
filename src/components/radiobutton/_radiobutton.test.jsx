import * as React from 'react';
import { customRender } from '../../../test/test-utils';
import Radiobutton from '.';

it('should render without throwing an error', () => {
  const { container } = customRender(<Radiobutton>Test</Radiobutton>);
  expect(container.firstChild).toMatchSnapshot();
});
