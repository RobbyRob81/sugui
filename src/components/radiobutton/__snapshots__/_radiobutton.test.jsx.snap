// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without throwing an error 1`] = `
<div
  aria-checked="false"
  class="sc-bdVaJa fVCJpx"
  role="radio"
  tabindex="0"
>
  <div
    class="sc-bwzfXH euzcvb"
    color="red"
  />
  <div
    class="sc-htpNat hSkRsC"
  >
    Test
  </div>
</div>
`;
